src/Returns/Http/Requests/ReturnRequest.php

```diff
@@ -33,7 +33,7 @@ public function returnAddress(): Address
         return Address::from($this->input('data.return_address'));
     }
 
-    public function payment(): ReturnPayment
+    public function payment(): ReturnPayment|null
     {
         return ReturnPayment::from($this->input('data.payment'));
```

Comment:
Let's add a little test to `ReturnRequestTest.php` to verify it can handle data without a payment key.
Because I'm not 100% sure `ReturnPayment::from(null)` will work and return `null` as well.